mscore/editstyle.cpp

```diff
@@ -718,21 +718,28 @@ void EditStyle::on_comboFBFont_currentIndexChanged(int index)
 
 void EditStyle::on_buttonTogglePagelist_clicked()
       {
-
-      if (pageList->isVisible()) {
-            pageList->setVisible(false);
```

Comment:
I see no difference setting `isVisible(false)` here or at the end of the function.  Maybe there used to be a reason to do it first,  before the other code, but I cannot find that reason now.  Visually, on the screen, it's the same.